app/pb_kits/playbook/pb_contact/_contact.jsx

```diff
@@ -26,15 +27,24 @@ const Contact = ({
   className,
   dark=false,
   contactValue,
+  contactDetail,
 }: ContactProps) => {
 
+  const formatDetail = (contactDetail) => {
+    if (contactDetail !== undefined) {
+      return `\u00b7 ${contactDetail}`
+    } else {
+      return ""
+    }
+  }
+
   const css = classnames(kitClasses({contactType}), className)
 
-  const formatPhoneNumber = (phoneNumberString, contactType) => {
+  const formatContact = (contactString, contactType) => {
     if (contactType == "email") {
-      return phoneNumberString
+      return contactString
     } else {
-      let cleaned = ('' + phoneNumberString).replace(/\D/g, '')
+      let cleaned = ('' + contactString).replace(/\D/g, '')
       let match = cleaned.match(/^(1|)?(\d{3})(\d{3})(\d{4})$/)
```

Comment:
Also not mutating `match` here so use `const`

```suggestion
      const match = cleaned.match(/^(1|)?(\d{3})(\d{3})(\d{4})$/)
```

also consider this variable name as the regex is only sort-of clear to readers. Maybe a more descriptive name that tells us `what we are matching` would be better here.

Reply:
Got it changed! Thanks Stephen.